src/jdk.jpackage/share/classes/jdk/jpackage/internal/resources/HelpResources_ja.properties

```diff
@@ -36,7 +36,7 @@ MSG_Help_win_install_dir=デフォルトのインストール場所の下の相
 --mac-signing-key-user-nameと組み合せることはできません。\n  --mac-installer-sign-identity <identity>\n          "pkg"インストーラの署名に使用するアイデンティティ。この値は直接\n          "productbuild"ツールの--signオプションに渡されます。このオプションは\n          --mac-signing-key-user-nameと組み合せることはできません。\n  --mac-app-store\n          jpackage出力がMac App Store用であること\n          を示します。\n  --mac-entitlements <file path>\n          バンドルの実行可能ファイルおよびライブラリの署名時に\n          使用する権限を含むファイルのパス。\n  --mac-app-category <category string>\n          アプリケーションのplistのLSApplicationCategoryTypeを生成する際に使用する文字列。\n          デフォルト値は"utilities"です。\n
-MSG_Help_linux_install=\  --linux-package-name <package name>\n          Linuxパッケージの名前。アプリケーション名にデフォルト設定されています\n  --linux-deb-maintainer <email address>\n          .debパッケージのMaintainer\n  --linux-menu-group <menu-group-name>\n          このアプリケーションが配置されているメニュー・グループ\n  --linux-package-deps\n          アプリケーションに必要なパッケージまたは機能\n  --linux-rpm-license-type <type string>\n          ライセンスのタイプ(RPM .specの"License: <value>")\n  --linux-app-release <release value>\n          RPM <name>.specファイルのリリース値または\n          DEBコントロール・ファイルのDebianリビジョン値。\n  --linux-app-category <category value>\n          RPM <name>.specファイルのグループ値または \n           DEBコントロール・ファイルのセクション値\n  --linux-shortcut\n          アプリケーションのショートカットを作成します。\n
+MSG_Help_linux_install=\  --linux-package-name <package name>\n          Linuxパッケージの名前。アプリケーション名にデフォルト設定されています\n  --linux-deb-maintainer <email address>\n          .debパッケージのMaintainer\n  --linux-menu-group <menu-group-name>\n          このアプリケーションが配置されているメニュー・グループ\n  --linux-package-deps <package-dep>[, <package-dep>...]\n          アプリケーションに必要なパッケージまたは機能\n  --linux-rpm-license-type <type string>\n          ライセンスのタイプ(RPM .specの"License: <value>")\n  --linux-app-release <release value>\n          RPM <name>.specファイルのリリース値または\n          DEBコントロール・ファイルのDebianリビジョン値。\n  --linux-app-category <category value>\n          RPM <name>.specファイルのグループ値または \n           DEBコントロール・ファイルのセクション値\n  --linux-shortcut\n          アプリケーションのショートカットを作成します。\n
```

Comment:
`--linux-package-deps <package-dep-string>` to sync with HelpResources.properties